ui/lib/core/addon/components/form-field.hbs

```diff
@@ -16,13 +16,13 @@
             <G.Legend data-test-form-field-label>{{this.labelString}}</G.Legend>
           {{/if}}
           {{#if this.helpTextString}}
-            <G.HelperText data-test-help-text>{{this.helpTextString}}</G.HelperText>
+            <G.HelperText data-test-form-field-help-text>{{this.helpTextString}}</G.HelperText>
```

Comment:
Because this helper text is not form field specific, meaning it could be seen in other components, I'd recommend changing this selector to the following and adding it under the FORMs area of General Selectors.
```suggestion
            <G.HelperText data-test-help-text={{@attr.name}}>{{this.helpTextString}}</G.HelperText>
```